Makefile

```diff
@@ -58,7 +58,7 @@ test.integration:
 
 
 update-requirements:
-	pip install pip-tools==6.1.0
+	pip install pip-tools==7.4.1
```

Comment:
Is there a reason for pinning `pip-tools`? 

`6.1.0` has a conflict with newer versions of `pip`, so I upgraded `pip-tools` instead. AFAICT, this make target is only used for development.

Reply:
i think the goal is consistency across all of our machines, if we were running different enough versions of the same tool, the result would probably diverge. 

To be honest though, I don't use this make command and I'm not sure if the other devs do, and I've just been installing pip-tools unpinned when I need it :/

Reply:
Cool thanks! I'll leave this pinned to a new version for now then.